### PR TITLE
Preprocessed description in Dataframe

### PR DESCRIPTION
**Pull request by cdelapena**
This PR adds a preprocessed_description field to the Dataframe file while leaving the original description field undisturbed.

- Adds preprocessed_description field to backend files make_db.py, models.py and sprocs.py.
- Passes job descriptions through preprocess_text function as part of the insert_new_sources function in backend/utils/sql/sql.py.
- Relocates text_preprocessing.py and reference_text.py files to the backend/utils/sql folder.
- Includes additional stopwords -> please, thanks, job, position.
- Replaces STRING_AGG with group_concat in sprocs.py file so that the code will run on my local machine. **Per https://www.sqlite.org/lang_aggfunc.html it looks like STRING_AGG is compatible with machines with PostgreSQL, while machines with MySQL (mine) are compatible with group_concat. Strange thing we may have to revisit if this causes issues from our hosted machine.

### CODE REVIEW

File: backend/sprocs.py [hunk 1]
```diff
@@ -34,8 +34,9 @@ def get_all_job_postings(db_filename) -> str:
                 company_name,
                 job_title,
                 description,
+                preprocessed_description,
                 salary,
-                STRING_AGG(t.name, ', ') AS tags,
+                group_concat(t.name, ', ') AS tags,
```
Comment (GomeChas) [inline backend/sprocs.py:39]: I like to keep method calls highly visible.  Can you cast to uppercase() on `GROUP_CONCAT`.

File: backend/sprocs.py [hunk 2]
```diff
@@ -48,10 +49,12 @@ def get_all_job_postings(db_filename) -> str:
                 company_name,
                 job_title,
                 description,
+                preprocessed_description,
                 salary,
                 job_type,
                 source_url,
                 publish_date
+            LIMIT 3
```
Comment (GomeChas) [inline backend/sprocs.py:57]: We should also not throttle this.

File: backend/sprocs.py (hunk 1 above)
Comment (cdelapena) [reply backend/sprocs.py]: GROUP_CONCAT uppercase() complete.

File: backend/sprocs.py (hunk 2 above)
Comment (cdelapena) [reply backend/sprocs.py]: Throttling removed.

**Review by GomeChas — APPROVED** [review]
All set, nice!

**Pull request merged**